topics/browser/02-dom/11-code-challenges/README.pt-BR.md

```diff
@@ -1,6 +1,6 @@
 # Desafios
 
-- Tipo: `exercício`
+- Tipo: `leitura`
```

Comment:
I think this is not really a problem with the content, but rather an issue with the LMS. I believe this part should be of type `practice` (`exercicio`), the same as in [spanish](https://github.com/Laboratoria/bootcamp/blob/master/topics/browser/02-dom/11-code-challenges/README.md#retos).

If I remember correctly, the [LMS should treat parts of type `practice` like a normal `read` when they do not include individual exercises with unit tests](https://github.com/Laboratoria/lms.laboratoria.la/blob/master/src/containers/unit.jsx#L75). In fact, the spanish version used to work, but no longer seems to be the case... :scream:

Esto parece ser un bug que afecta a un montón de partes en diferentes cursos!!!! Por ejemplo: https://lms.laboratoria.la/cohorts/test/courses/javascript/01-basics/04-guided-exercises

cc/ @Xpktro @MaiaRojas @RuthMeryCardenas

Reply:
In the end this was identified as an issue in `curriculum-parser` and not the content or LMS. This issue has now been fixed, so these parts will be changed back to `type: 'exercício'`.